tdrs-backend/tdpservice/users/test/test_api/test_roles.py

```diff
@@ -71,28 +71,10 @@ def test_role_update_not_found(api_client, create_test_users):
     assert response.status_code == status.HTTP_404_NOT_FOUND
 
 
-# @pytest.mark.django_db
-# def test_role_update_unauthorized(api_client, create_test_users):
-#     """Test data analyst does not have access."""
-#     group = Group.objects.get(name="Data Analyst")
-#     api_client.login(username="test__data_analyst", password="test_password")
-#     response = api_client.patch(f"/v1/roles/{group.id}/", {"name": "staff"})
-#     assert response.status_code == status.HTTP_403_FORBIDDEN
-
-
 @pytest.mark.django_db
 def test_role_delete_not_found(api_client, create_test_users):
     """Test role deletion no longer exists."""
     group = Group.objects.get(name="OFA Admin")
     api_client.login(username="test__ofa_admin", password="test_password")
     response = api_client.delete(f"/v1/roles/{group.id}/")
     assert response.status_code == status.HTTP_404_NOT_FOUND
-
```

Comment:
@jtwillis92 As we discussed, let's add three tests to outline the permissions for our three roles. These should form a simple template for what @riatzukiza will need for his Regional testing.

Reply:
Added here: tdrs-backend/tdpservice/users/test/test_permissions.py